utils/src/main/kotlin/storage/FileArchiver.kt

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2019 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.utils.storage
+
+import com.here.ort.utils.log
+
+import java.io.File
+import java.nio.file.FileSystems
+import java.nio.file.Paths
+
+/**
+ * A class to archive files matched by provided [patterns] in a [FileStorage][storage].
+ */
+class FileArchiver(
```

Comment:
I'm not sure about the name... when I read "archiver" I thought you mean Zip etc. But I also cannot come up with a better name right now.

Reply:
I agree that could be misleading, but I also have no better idea for the name right now.

Reply:
`Backupper`? :)

Reply:
Or we call the class `FileStore` (also see [storage vs store](https://wikidiff.com/storage/store)) and the function `store`.

Reply:
As discussed offline I have renamed the class to `FileSaver`.